Modules/xASL_module_ASL.m

```diff
@@ -187,6 +188,22 @@
 	x.Q.NumberEchoTimes = 1;
 end
 
+%% D6. DeltaM parsing - check if all/some volumes are deltams
+% If TSV file exist
+[pathDir, pathASL4D] = fileparts(x.P.Path_ASL4D);
+pathASL4Dcontext = fullfile(pathDir, [pathASL4D '_aslcontext.tsv']);
+% We don't have a subtraction image by default
+x.modules.asl.bDeltaM = 0;
+if xASL_exist(pathASL4Dcontext,'file')
+	% Load TSV file
+	aslContext = xASL_tsvRead(pathASL4Dcontext);
+	bidsPar = xASL_bids_Config;
+	% Check for presence of deltaM subtraction volumes
+	if numel(regexpi(strjoin(aslContext(2:end)),bidsPar.stringDeltaM)) > 0
```

Comment:
Can we not find a better name for `stringDeltaM`? Is this supposed to contain a string with all `deltaM` volumes? Why not logic?

And if you have at least 1 deltaM (according to this `if` statement), does this mean that we assume that all `ASL4D` volumes are `deltaM`? In that case I would add a warning (unless vendor is GE perhaps)

Reply:
this you got wrong. `bidsPar` is a function and variable where we set all the default variable and tag names for BIDS and also set all the dependencies so that they are configured in a single file and easily resolved.

So `bidsPar.stringDeltaM` contains the string `deltam` so that in case this is changed in BIDS, we only have to change it in the `xASL_bids_Config` ? That OK? This has been there for half a year ;)

If we have a mixture of control/label/deltaM, then we are doomed - we don't really support these more complex stuff. Lets add a warning and implement late?

Reply:
The mixture of `control-label-deltaM` is realistic. I'm fine with all this, as long as we add proper warnings in place!

Reply:
The mix wouldn't be correct according to BIDS AFAIK, so the validation should already alert you. Right now we still recommend to run the BIDS validation. We don't say anywhere that we do perfect validation ourselves. If we start validating that stuff from start to finish, there are a hundred new issues you can theoretically open. That's not really part of this PR IMHO.

Reply:
Vendors do all crazy things of putting `control` and `label` and `deltaM` and `cbf` in the same sequence. Besides, it is not strictly against BIDS definition to have all these there. I don't foresee why would anybody have some controls and some deltams in the same sequence that wouldn't be simply duplicite. But that's nowhere in our Flavors. I would thus recommend to make this BIDS check in the import and not here. I've made a new issue for that. One of the reasons why not to do it now is that we don't have any such flavors, we therefore don't know what to expect and how to properly react to that.

Reply:
Sounds good